stable/postgresql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: postgresql
-version: 0.18.1
+version: 0.18.2
```

Comment:
This being a new feature, it may require a minor version change.

Reply:
Changed to 0.19.0